mod/metrics/set/abstract/value/edit.rb

```diff
@@ -38,8 +38,8 @@ def editor
   def edit_fields
     return if voo.editor == :standard
     [
-      [card, { title: "Answer", editor: :standard }],
-      [unknown_field_card, { hide: :title }],
+      [card, { title: "Answer", editor: :standard, hide: :help }],
```

Comment:
oh, here we're using "Answer" as the title of the value card, and above we're using it as the title of the whole answer?

My developer brain says the Answer should be the Answer and the Value the value, but having both answer and value is probably confusing, so I guess this is best.

...but my developer brain needed to talk to another developer brain ;)